src/App.jsx

```diff
@@ -1,3 +1,40 @@
+import { BrowserRouter as Router, Switch, Route,  } from 'react-router-dom';
+import Home from './views/Home';
+import SignIn from './components/SignIn';
+import Profile from './views/Profile';
+import SignUp from './components/SignUp';
+import PrivateRoute from './components/PrivateRoute';
+import { UserProvider } from './context/UserContext';
+import { ProfileProvider } from './context/ProfileContext';
+
+import EditProfile from './components/EditProfile';
+import Header from './components/Header';
```

Comment:
SignIn, SignUp, and EditProfile are all views instead of components. Views are essentially the pages of the site.